tests/unittests/tests-hashes/tests-hashes-sha384-hmac.c

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (C) 2016 Martin Landsmann <martin.landsmann@haw-hamburg.de>
+ * Copyright 2016 OTA keys S.A.
```

Comment:
Please update the copyright accordingly.

Reply:
Sorry for the wait, done.